build/importer_engine_support_check.sh

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
```

Comment:
If we keep this script around I think it will be useful to convert this to Go at some point for maintainability.

Reply:
What's the best practice for where to put a .go file like this? I don't want it to be a test because it shouldn't run every time, but it doesn't seem right to put it under cmd/.

Reply:
I am fine with adding a top level /scripts folder or under build/scripts.

https://github.com/golang-standards/project-layout/tree/master/scripts has some guidance.

Reply:
Thanks. I'll move the file to go and put it under `scripts/`.

Reply:
Done.